src/templates/vibso_classes.tsv

```diff
@@ -14,10 +14,7 @@ obo:VIBSO_0000010	groove density setting	grating|grating setting	https://github.
 obo:VIBSO_0000011	diffraction grating		https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/49	A device that is an optical component with a periodic structure that diffracts light into several beams travelling in different directions (i.e. different diffraction angles)	https://en.wikipedia.org/wiki/Diffraction_grating	owl:Class	device	ORCID:0000-0002-1595-3213			This class should better be defined in a more general device ontology or a domain ontology with the scope of optical devices.			
 obo:VIBSO_0000012	deprecated_excitation wavelength		https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/48	A physical quality of the light source that is used used to excite a system to a higher energy state.		owl:Class		ORCID:0000-0002-1595-3213			This class is not needed, as PATO:wavelength is sufficient, if this quality of a Raman device is really needed.	true	failed exploratory term	wavelength
 obo:VIBSO_0000013	excitation wavelength setting		https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/48	A setting datum that specifies the wavelength of an electromagnetic radiation source (e.g. a laser) which has the function to excite the energy state of a target material.		owl:Class	setting datum	ORCID:0000-0002-1595-3213			This class should better be defined in a more general device ontology or a domain ontology with the scope of optical devices.			
-obo:VIBSO_0000014	objective magnification factor setting	magnification|magnification setting	https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/24 	A setting datum that specifies the magnification factor of the objective collecting the scattered light.		owl:Class	setting datum	ORCID:0000-0002-1595-3213|ORCID:0000-0001-7694-5519|ORCID:0000-0002-2239-3955	"example:RamanSpectrometerX rdf:type obo:CHMO_0001235 (Raman spectrometer) ;
-            obo:OBI_0000293 (has_specified_input) example:objective_magnification_factor_setting_for_RamanSpectrometerX .
-example:objective_magnification_factor_setting_for_RamanSpectrometerX rdf:type obo:VIBSO_0000014 ;
-            obo:OBI_0002815 (has representation) '100 x' ."					
+obo:VIBSO_0000014	objective magnification factor setting	magnification|magnification setting	https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/issues/24 	A setting datum that specifies the magnification factor of the objective collecting the scattered light.		owl:Class	setting datum	ORCID:0000-0002-1595-3213|ORCID:0000-0001-7694-5519|ORCID:0000-0002-2239-3955	100x
```

Comment:
We agreed to simplify this example to be in line with the other examples after the linebreaks were accidentally removed in [a484e11](https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/pull/133/commits/a484e1134afb37982a1f3496ed6c2e6e75e3e9d7) by the spreadsheet editor.